common/scala/src/main/scala/whisk/http/PoolingRestClient.scala

```diff
@@ -131,7 +108,7 @@ class PoolingRestClient(
   }
 
   // Runs a request and returns either a JsObject, or a StatusCode if not 2xx.
-  protected def requestJson[T: RootJsonReader](futureRequest: Future[HttpRequest]): Future[Either[StatusCode, T]] = {
+  def requestJson[T: RootJsonReader](futureRequest: Future[HttpRequest]): Future[Either[StatusCode, T]] = {
```

Comment:
I can't put a comment there, so leaving it here:

Would it make sense to rename `request0` to `request`? Not sure why it was `request0` in the first place and I think from an interface standpoint `request` makes sense? WDYT?